proto/exocore/avs/tx.proto

```diff
@@ -30,39 +29,15 @@ message AVSInfo {
     (gogoproto.nullable) = false
   ];
   //avs epoch ,Subsequently  will be handled by the epochs module.
-  EpochInfo avs_epoch = 8;
+  string epoch_identifier = 8;
   // registered operator of avs
   repeated string operator_address = 9;
-}
 
-// EpochInfo defines the message interface containing the relevant informations aboutt
-// an epoch.
-message EpochInfo {
-  // identifier of the epoch
-  string identifier = 1;
-  // start_time of the epoch
-  google.protobuf.Timestamp start_time = 2
-  [(gogoproto.stdtime) = true, (gogoproto.nullable) = false, (gogoproto.moretags) = "yaml:\"start_time\""];
-  // duration of the epoch
-  google.protobuf.Duration duration = 3 [
-    (gogoproto.nullable) = false,
-    (gogoproto.stdduration) = true,
-    (gogoproto.jsontag) = "duration,omitempty",
-    (gogoproto.moretags) = "yaml:\"duration\""
-  ];
-  // current_epoch is the integer identifier of the epoch
-  int64 current_epoch = 4;
-  // current_epoch_start_time defines the timestamp of the start of the epoch
-  google.protobuf.Timestamp current_epoch_start_time = 5 [
-    (gogoproto.stdtime) = true,
-    (gogoproto.nullable) = false,
-    (gogoproto.moretags) = "yaml:\"current_epoch_start_time\""
-  ];
-  // epoch_counting_started reflects if the counting for the epoch has started
-  bool epoch_counting_started = 6;
-  // current_epoch_start_height of the epoch
-  int64 current_epoch_start_height = 7;
+  // Effective current epoch, accounting for current_epoch + 1
+  // and current_epoch is the integer identifier of the epoch module
+  int64 effective_current_epoch = 10;
```

Comment:
Can you rename this to something simpler like `starting_epoch` ?

Reply:
fixed